WatchUI/keywords/Tesseract.py

```diff
@@ -0,0 +1,109 @@
+import os
+from typing import Any, Union
+import cv2 as cv
+import pytesseract
+from ..Ibasic.IBasic import IBasic
+
+
+class Tesseract(IBasic):
+    """Class representing Tesseract and its methods.
+    This class is not instantiated itself, but inherited into main WatchUI module class.
+    Args:
+        Basics (object): interface class
+    """
+
+    def image_to_string(
+        self, path: str, oem: str="3", psm: str="3", language: str="eng", path_to_tesseract= ""
+    ) -> Union[bytes, str]:
+        """Converts provided image to string and returns.
+        Args:
+            path (str): path to the image
+            oem (str): [description]
```

Comment:
add description of the arguments please. I know, that I wrote this code, but I have no idea, what those arguments actually mean :)

Reply:
Done